### PR TITLE
Refactoring the project structure

### PR DESCRIPTION
**Pull request by adrian61**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,296 @@
...
+### PyCharm+all ###
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+# User-specific stuff
+.idea/**/workspace.xml
+.idea/**/tasks.xml
+.idea/**/usage.statistics.xml
+.idea/**/dictionaries
+.idea/**/shelf
```
Comment (Ard2148810) [inline .gitignore:56]: Why so much garbage in .gitignore?

File: backend/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from backend.library.SimpleWebSocketServer import *
+
+name="backend"
```
Comment (Ard2148810) [inline backend/__init__.py:3]: This whole frontend/backend naming convention is confusing. Backend doesn't equal server and frontend doesn't equal client.

**Review by Ard2148810 — APPROVED** [review]
It is even worse! That's why I like it.

File: .gitignore (hunk 1 above)
Comment (adrian61) [reply .gitignore:56]: Created automatically by gitignore.io

**Pull request merged**